templates/go/events/handle.go

```diff
@@ -10,18 +10,18 @@ import (
 
 // Handle a CloudEvent.
 // Supported Function signatures:
-//   func()
-//   func() error
-//   func(context.Context)
-//   func(context.Context) error
-//   func(cloudevents.Event)
-//   func(cloudevents.Event) error
-//   func(context.Context, cloudevents.Event)
-//   func(context.Context, cloudevents.Event) error
-//   func(cloudevents.Event, *cloudevents.EventResponse)
-//   func(cloudevents.Event, *cloudevents.EventResponse) error
-//   func(context.Context, cloudevents.Event, *cloudevents.EventResponse)
-//   func(context.Context, cloudevents.Event, *cloudevents.EventResponse) error
+// * func()
+// * func() error
+// * func(context.Context)
+// * func(context.Context) error
+// * func(event.Event)
```

Comment:
I see that cloudevents->event, is that correct given we use renaming import?

Reply:
> I see that cloudevents->event, is that correct given we use renaming import?

We could just change the import to `event`, right?

Reply:
You are right, that should be unified.  I like changing the import to shorter `event`.

Reply:
Updated